Assets/Tests/Runtime/Host/NetworkIdentityTests.cs

```diff
@@ -172,7 +172,7 @@ public void RemoveclientAuthorityNotSpawned()
         [Test]
         public void RemoveClientAuthorityOfOwner()
         {
-            serverObjectManager.AddCharacter(server.LocalPlayer, gameObject);
+            serverObjectManager.ReplaceCharacter(server.LocalPlayer, server.LocalClient, gameObject);
```

Comment:
`server.LocalClient` doesn't seem relevant to this test, maybe use default or null instead

Reply:
this will get addressed via: #774